src/components/header/header.jsx

```diff
@@ -44,8 +44,9 @@ var header = React.createClass({
 
                     this.props.isAdmin(data.user);
                 }
-
-                this.props.setUser(data.user);
+                if (this.props.setUser) {
```

Comment:
the setUser atribute was not being passed to all components and was therefore causing an error at times, you were at the morning party when it happened and it was only a fix untill i asked you about it because i didn't want to mess with the components. Totally forgot about it though :(